### DISCUSSION TITLE
Moffatt All The Things

### DISCUSSION THREAD

**Discussion by openenglishbible** [openenglishbible/Open-English-Bible#348] [Announcements]
Progress on the OEB has been a little bit slow for a while - we started with a bang with the NT but progress on the OT has been slow.

The number one reason for this is that for the NT I could start from a solid if slightly old fashioned base - the 20th Century New Testament, and update. This has yielded good results, at least in my opinion the English of the OEB is more natural  than, for example, the WEB which started from the deliberately archaic American Standard Version.

However when we came to the OT I immediately ran into the issue that there was no good base. Starting with the ASV would be pointless - it would be duplicating the WEB's excellent work and I don't have any reason to believe I would do a better job than they have. Translating the entire OT from scratch requires time and skill I don't have. So far I have been using instead a patchwork of source materials, primarily two early 20th Century OT scholars, John McFadyen and Charles Foster Kent, but it has been slow going.

However time marches on, and what wasn't possible when I started is now possible: we can use the translation by James Moffatt. Moffatt died in 1944 (ie greater than 70 years ago), so he is out of copyright in Europe and Australia, and his OT translation was published in 1924-26 (ie greater than 95 years ago) so he is now out of copyright in the US.

I would describe Moffatt's translation as modern in phasing if sometimes idiosyncratic. Updating it would be mostly a matter of conforming specific terms to harmonise with today's common usage, and conforming to the Masoretic text wherever possible in line with the general OEB aims - rather than a matter of completely rewriting.

I'm currently doing a revision of Joshua as a testbed of this approach and should get it uploaded in a couple of weeks. If all goes well, I think we should be able to kick into high gear on completing the OT.

**Comment by Coburn** [comment]
I'm only an occasional contributor, but I'm cheering for you. Sometimes a
one-man translation brings out things that a committee translation will
bury in the geniza of timidity.

If you have a minute, you might take a look at my translation of Amos
<https://www.academia.edu/42916839/Amos_A_Literary_Translation> and of First
John
<https://www.academia.edu/45477213/First_John_A_new_translation_from_a_proposed_Hebrew_original>,
simply to see the way that I try to walk the line between accuracy and
natural English. Just as an example. I've been told my English is a bit too
relaxed for the OEB, at least in those samples, and I don't know how to get
around it. Maybe just recommending it to you as another one-man
translation, just for a quick comparison, will be enough.

Thanks,

Coby

**Comment by vonlein** [comment]
I love Amos too. Took a stab at a couple chapters a few years ago. Would be interesting to compare our choices. Mine is a bit more intentionally formal/archaic in tone. I’m going to have to check out your document. 

https://www.academia.edu/44383753/Words_of_Amos_Literal_and_Literary_Translation_ch_1_2_

It’s funny when people think some of my work is a complete paraphrase just because it doesn’t seem like what they’ve seen before. So much of Bible translation is stuck in a rut based on repeating the same assumptions.

For instance, Genesis chapter 1: https://www.dropbox.com/s/7a7w3ezr9mki3v7/PsalmOfCreation.pdf?dl=0
And chapter 2: https://www.dropbox.com/s/ls2f02kiu52sq3r/TheEdenNarrative.pdf?dl=0
Or chapter 11: https://www.dropbox.com/s/594n3ddzezbutsj/PrimordialHistory6-TheTower-handout.pdf?dl=0

**Comment by prust** [comment]
@openenglishbible: Thank you for this guidance! Is https://archive.org/details/the-old-testament-vol.-2-james-moffatt-1925 the best source for an OCR of Moffatt's text? I looked, but didn't find any python scripts, regexes, etc, for converting OCR to USFM, separating uk/us spellings, etc -- please let me know if anything like this exists (no problem if it doesn't).

UPDATE: After looking more at the OCR from archive.org and from google books (looking at the first couple of Psalms, in particular), I'm starting to wonder if simply retyping would be faster and less error prone than correcting the OCR.

Update 2: Looks like https://github.com/openenglishbible/usfm-bibles/tree/master/Moffat is the best place to put original Moffatt transcriptions.

**Reply by openenglishbible** [reply]
Hi @prust, Either that one or https://archive.org/details/oldtestamentnewt0000unse should be fine. Main thing is not to go from a revised version, or from the combined OT+NT which came out in 1935.

Sadly I don't have a clean way of OCRing the PDF, the combination of the two column layout, old faded paper and chapter/verse numbers makes Bibles hard to scan cleanly!

**Reply by prust** [reply]
@openenglishbible:

> https://archive.org/details/oldtestamentnewt0000unse

That's a bit clearer & better quality, thanks! And thanks for the warning re: the revised version.

**Comment by WmWhitt** [comment]
Hello Prust--I just discovered the Open English Bible and am very impressed with the work you've done. I am currently most of the way through a project to translate the books of the Torah and the Former Prophets, and all my translations are available on my webpage at academia.edu (https://duke.academia.edu/WilliamWhitt) and my creator page on archive.org (https://archive.org/search?query=creator%3A%22William+Whitt%22&sort=-date). My translation of Deuteronomy is open access so you are free to use that as a companion to Moffatt if you had any interest in doing so. My other translations are fully copyrighted but if you are using Moffatt as a base, I think it would be fine to borrow from those translations in places where you think it made sense to. And I'd be happy to be a resource for you if you want another pair of eyes to check any of the Hebrew Bible translations you're doing. By the way, I skimmed parts of the Psalms translation and it is really outstanding--it captures the rhythm and force of the Hebrew quite well.

**Reply by prust** [reply]
Thanks @WmWhitt but I'm actually quite new to the scene here. I've been (slowly) working away at transcribing Moffatt's translation of the Psalms in https://github.com/openenglishbible/usfm-bibles/pull/3, but that's all I've done. Russell Allen (@openenglishbible) has been leading the charge with the OEB and has put the most work into it over the years.

**Comment by WmWhitt** [comment]
Thanks Peter for clearing up my confusion! I'll send a note to Russell. The
Moffatt translation of the Psalms is really well done. I am somewhat
familiar with his translation, but it has been quite a few years since I've
looked at it, and I was really surprised by how well he captured the tone
of the Hebrew.
Best,
Bill

**Comment by foolishgrunt** [comment]
@russellallen Could you still use help transcribing Moffatt? I have few relevant technological skills to contribute (other than what I picked up as I taught myself Linux) and no expertise at all in Greek/Hebrew, but transcribing documents is definitely something I could handle. If so, please let me know if there's a book that could especially benefit from the attention.

**Comment by Coburn** [comment]
Is the transcription something that could be helped by using OCR? I have
had good luck with gimagereader.

**Reply by foolishgrunt** [reply]
Archive.org already has a pretty good OCR (https://archive.org/stream/oldtestamentnewt0000unse/oldtestamentnewt0000unse_djvu.txt), I suspect most of the remaining work would be in visually verifying and formatting.

**Comment by prust** [comment]
To update this thread: @jbj99 has been helping me with transcribing Psalms (I've transcribed Psalms 1-55 so far, he's started at Psalm 100) and he has also worked on Obadiah and Haggai. You can find his work here: https://github.com/jbj99/usfm-bibles/tree/master/Moffat, and mine is here: https://github.com/openenglishbible/usfm-bibles/pull/3. If anyone wants to jump in and help with transcribing the Minor Prophets, you're more than welcome, you'll just want to communicate with @jbj99 to avoid duplicating work.

@foolishgrunt: I'm sorry no one responded to your question re: help transcribing -- I think everyone on this project is volunteering their time, and sometimes people aren't available due to other things in life. But I do think we could use the help with transcribing, if you're still available. The main thing is just to avoid duplicate work by communicating on this thread what you're working on (and ideally putting it on Github so that everyone can see your progress -- this is especially helpful if your work gets stalled and someone else wants to pick up where you left off).

Even if OCR did work perfectly in capturing the english words, it won't properly separate the verse numbers or properly capture the [poetry indentation in USFM format](https://ubsicap.github.io/usfm/poetry/index.html). So at least for the Psalms (which has a lot of staggered indentation), transcribing wouldn't take much longer than properly tagging verse numbers and indentation of a perfect OCR. But as @openenglishbible mentions above, the two column layout, faded paper, etc, makes OCR anything but perfect. My experience with it -- at least the OCR that archive.org uses -- is that it is *much* faster to transcribe than to fix all the OCR issues. Of course everyone's free to use whatever tools they want in helping with digitizing Moffatt (and if anyone has a better experience with OCR than I did, I'm all ears), I just wanted to share my experience.

**Reply by prust** [reply]
Update: @jbj99 has started transcribing Hosea.

**Reply by jbj99** [reply]
Yes, I've started tinkering in Hosea while we finish up the psalms. 
At the moment, my plan for the minor prophets is to work through them in order (so after Hosea, Joel, Amos...). I'll try to remember to post an update here when I start on a new book.

**Reply by foolishgrunt** [reply]
Thanks @prust for responding. Unfortunately, for the time being all of my after-work-hours volunteer time is being taken up by another project I've undertaken. But in case that should change, I'll definitely stay tuned here and notify y'all if I start upon a book.

**Comment by jbj99** [comment]
@openenglishbible - now that @prust and I have finished transcribing the psalms, are there any particular book/priorities that you would like me to work on transcribing from Moffat? Otherwise I may continue with the minor prophets, then whichever other books take my fancy after that 😂
I have started on Joel and plan to continue to continue through the minor prophets in order, unless there are other priorities.
